CONTRIBUTING.md

```diff
@@ -1,58 +1,79 @@
-
-HDX developers are using [VS Code](https://code.visualstudio.com/) as a standard IDE for this project with development taking place inside Docker containers.
-
-## Notes:
-For **development purposes** the project assumes that the HAPI stack is running. Or more precisely the HAPI db container.
-One can see that the [Docker Compose configuration](https://github.com/OCHA-DAP/hdx-hapi-write-app/blob/cc75b0c567fd40016e37a4599942e8c8769bdb24/docker/docker-compose.yml#L40) 
-refers to the network `hapi_stack_hapi`. 
-In this way, any change to the database will be reflected in the locally running HAPI API endpoints.
-
-For **testing purposes** there is a test database container that can be started. 
-It's only meant to start under the [testing profile](https://github.com/OCHA-DAP/hdx-hapi-write-app/blob/cc75b0c567fd40016e37a4599942e8c8769bdb24/docker/docker-compose.yml#L35).
-`docker-compose --profile=testing up -d`
-
-
-## Starting the project:
-First start the HAPI project. So run the following in the HAPI project folder:
-
-```shell
-cd docker
-docker-compose start
-```
-Then spin up the HWA project. So run the following in the HWA project folder:
-
-```shell
-cd docker
-docker-compose up -d
-docker-compose exec -T hwa sh -c "apk add git"
-docker-compose exec -T hwa sh -c "pip install --upgrade -r requirements.txt"
-docker-compose exec -T hwa sh -c "pip install --upgrade -r dev-requirements.txt"
-cd ..
-```
-
-To start the project: 
-```shell
-cd docker
-docker-compose exec -T hwa sh -c "python start.py"
-cd ..
-```
-
-If one also wants to run the tests:
-```shell
-./initialize_test_db.sh
-cd docker
-docker-compose exec -T hwa sh -c "pytest --log-level=INFO"
-cd ..
-```
-
-## Project structure
--  `hdx_hwa.config` - everything to do with runtime configuration
--  `hdx_hwa.db` - data access layer
-   -  `hdx_hwa.db.services` - functions, objects that need to be used outside of `hdx_hwa.db`
--  `hdx_hwa.engine` - execution engine for a JSON patch
-   -  `hdx_hwa.engine.services` - functions, objects that need to be used outside of `hdx_hwa.engine`
--  `hdx_hwa.patch_repo` - new patch discovery, communication with the patch repo
+# Contributing 
+
+The HAPI Write App (HWA) is being developed by a team from the [Centre for Humanitarian Data](https://centre.humdata.org/).
+
+HDX developers are using [VS Code](https://code.visualstudio.com/) as a standard IDE for this project with development taking place inside Docker containers.
+
+## One-off container creation activities
+
+There is a test database container that can be started without the HAPI stack which needs to be created in one-off setup. It's only meant to start under the [testing profile](https://github.com/OCHA-DAP/hdx-hapi-write-app/blob/cc75b0c567fd40016e37a4599942e8c8769bdb24/docker/docker-compose.yml#L35)
+
+To create the HWA stack containers in the first instance, including a test database container:
+
+```shell
+docker network create -d bridge hapi_stack_hapi

```

Comment:
Just a note: I think that creating the network manually is only needed in case one hasn't setup the `hdx-hapi` docker stack before. In other words, the network should be created automatically when running `docker compose up -d` in `hdx-hapi`

Reply:
Yes, but it is useful in that it indicates this network is required and the command is not destructive.